main.sh

```diff
@@ -59,7 +59,7 @@ cluster_up() {
 	run cp "$kubeconfig" "$CLUSTER_KUBECONFIG"
 	export KUBECONFIG="$CLUSTER_KUBECONFIG"
 
-	if is_set "${PROMETHEUS_ENABLE}" || is_set "GRAFANA_ENABLE"; then
+	if is_set "$PROMETHEUS_ENABLE" || is_set "$GRAFANA_ENABLE"; then
```

Comment:
this is significant .. don't know how it was missed. 
We should validate the usecases in CI

Reply:
I have filed - https://github.com/sustainable-computing-io/local-dev-cluster/issues/42